lebedigital/calibration/forwardmodel_linear_elastic_cylinder.py

```diff
@@ -23,7 +22,7 @@ def definition(self):
                               Sensor("height"),
                               Sensor("radius"),
                               Sensor("displacement_list")]
-        self.output_sensors = [Sensor('force_list')]
+        self.output_sensors = [Sensor('force_list', std_model="sigma")]
```

Comment:
what is std_model? @eriktamsen what is the height here (30cm or 10cm)?

Reply:
see above

Reply:
It was part, necessary for the version of ProbEye, I dont remember why.